src/FunctionsFramework.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Copyright 2021 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\CloudFunctions;
+
+use Google\CloudFunctions\Registry;
+
+class FunctionsFramework
+{
+    public static function http(string $name, callable $fn)
+    {
+        Registry::registerHttp($name, $fn);
+    }
+
+    public static function cloudEvent(string $name, callable $fn)
```

Comment:
I think in other languages we treated `cloudevent` as though it is all one word, and thus the identifier is all lower case. I can't remember which way is correct, but @grant probably does. We should be consistent across languages.

Reply:
Oh, you're right we did one word in python and node. I think it should be two words as the official SDKs and their types use `CloudEvent`.  I checked Java, Ruby, Go, and .NET and those are all using two word versions already.

Reply:
OK we should go back and update Nodejs and Python

Reply:
FYI @KaylaNguyen

Reply:
I filed bugs:
https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/378
https://github.com/GoogleCloudPlatform/functions-framework-python/issues/166